PluginTemplateProject/pom.xml

```diff
@@ -131,6 +131,11 @@
                 <groupId>org.scala-tools</groupId>
                 <artifactId>maven-scala-plugin</artifactId>
                 <version>2.15.2</version>
+                <configuration>
+                    <excludes>
+                        <exclude>com/alpine/templates/db/*</exclude>
+                    </excludes>
+                </configuration>
```

Comment:
I guess the way I see this template is not that it is ever really included in its current form as an operator.  Instead, you copy the template into your real directory, modify it, add it to the pom.xml and test it out.  If they want to then create another operator, they can do the same way.  The original template code remains to recopy each time you start a new operator, but is inert.

That's reflected in the //todos built into the template
//todo - copy this template into your your own package and change the package name
//todo - rename Signature, GUINode and Runtime classes to refer to your operator
//todo - update plugins.xml to include your new Signature name